src/main/java/Duke.java

```diff
@@ -1,10 +1,147 @@
+        command = in.nextLine();
+        while(!command.equals("bye")){
+            if(command.equals("list")) printList(tasks, tasksTotal);
+            else if(command.contains("done")){
+                int index = Integer.parseInt(command.substring("done".length()+1));
+                taskToMark = tasks[index-1];
+                taskToMark.markedAsDone();
+            }
+            else if(command.contains("todo")||command.contains("deadline")||command.contains("event")){
+                if (command.contains("todo")) {
+                    String taskTodo = command.substring("todo".length()+1);
+                    taskToAdd = new Todo(taskTodo);
+                }
+                if(command.contains("deadline")){
+                    int position = command.indexOf("/");
+                    String taskDeadline = command.substring("deadline".length()+1,position-1);
+                    String by = command.substring(position+"/by".length()+1);
+                    taskToAdd = new Deadline(taskDeadline);
+                    ((Deadline) taskToAdd).setBy(by);
+                }
+                if(command.contains("event")){
+                    int position = command.indexOf("/");
+                    String taskEvent = command.substring("event".length()+1,position-1);
+                    String at = command.substring(position+"/at".length()+1);
+                    taskToAdd = new Event(taskEvent);
+                    ((Event) taskToAdd).setAt(at);
+                }
+                tasks[tasksTotal]=taskToAdd;
+                tasksTotal++;
+                printAddMessage(taskToAdd);
+                printNumOfTasksInList(tasksTotal);
+            }
+            command = in.nextLine();
+        }
+        printExitMessage();
+
+
+    }
+    public static void printExitMessage(){
+        printLine();
+        System.out.println("     Bye. Hope to see you again soon!");
+        printLine();
+    }
+    public static void printWelcomeMessage(){
+        printLine();
+        System.out.println("     Hello！ I'm Duke");
+        System.out.println("     What can I do for you?");
+        printLine();
+    }
+    public static void printLine(){
+        System.out.println("    ____________________________________________________________");
+    }
+    public static void printAddMessage(Task task){
+        printLine();
+        System.out.println("     Got it. I've added this task:");
+        System.out.println("      "+task);
+    }
+    public static void printNumOfTasksInList(int tasksTotal){
+        if(tasksTotal==1) System.out.println("     Now you have 1 task in the list.");
+        else System.out.println("     Now you have "+tasksTotal+" tasks in the list.");
+        printLine();
+    }
+    public static void printList(Task[] tasks,int tasksTotal){
+        printLine();
+        System.out.println("     Here are the tasks in your list:");
+        for(int i=0;i<tasksTotal;i++){
```

Comment:
You might want to include a space after each punctuation in the for loop so for example, for(initialization; condition; update), there is a space between ; and condition as well as ; and update